backend/conferences/types.py

```diff
@@ -1,88 +1,41 @@
 from datetime import datetime
+from typing import List
 
-import graphene
 import pytz
-from graphene_django import DjangoObjectType
+import strawberry
+from api.scalars import Date, DateTime
 from languages.types import LanguageType
-from schedule.types import ModelScheduleItemType
+from schedule.types import RoomType, ScheduleItemType
 from submissions.types import SubmissionType, SubmissionTypeType
 
-from .models import AudienceLevel, Conference, Deadline, Duration, TicketFare, Topic
 
+@strawberry.type
+class AudienceLevelType:
+    id: strawberry.ID
+    name: str
 
-class DurationType(DjangoObjectType):
-    allowed_submission_types = graphene.NonNull(
-        graphene.List(graphene.NonNull(SubmissionTypeType))
-    )
 
-    def resolve_allowed_submission_types(self, info):
-        return self.allowed_submission_types.all()
+@strawberry.type
+class TopicType:
+    id: strawberry.ID
+    name: str
+
+
+@strawberry.type
+class ConferenceType:
```

Comment:
Same as above, I think it might nice to remove Type from all the strawberry types :)